baus/datasources.py

```diff
@@ -886,13 +875,18 @@ def superdistricts_geography():
 
 
 @orca.table(cache=True)
-def sqft_per_job_adjusters(): 
-    return pd.read_csv(os.path.join(misc.configs_dir(), "adjusters", orca.get_injectable("sqft_per_job_adj_file")), index_col="number")
+def sqft_per_job_adjusters(run_setup):
+    # Add if statement in case the file is not specified
+    if run_setup["sqft_per_job_adj_file"] is not None:
```

Comment:
A little off topic relative to PR, but while we are here: efficiency of not having a separate orca step is definitely more direct, and the conditional means we wont get an error on import - though it may come later if `sqft_per_job_adjusters=True` and a file is not found.